app/templates/buyers/_base_edit_question_page.html

```diff
@@ -53,7 +53,18 @@ <h1 class="govuk-heading-l">
               </span>
             {% endif %}
             {% for question in question.questions %}
-              {% if errors and errors[question.id] %}
+              {% set is_govuk_frontend = govuk_frontend_from_question(question) %}
+              {% if is_govuk_frontend %}
+                {% set govuk_forms = {"govukInput": govukInput, "dmListInput": dmListInput} %}
+                {% set form = govuk_frontend_from_question(question, brief, errors) %}
+                {%- if form.label %}
+                  {{ govukLabel(form.label) }}
+                {% endif -%}
+                {% if not form.params.question_advice %}
+                  {{ question.question_advice }}
+                {% endif %}
+                {{ govuk_forms[form.macro_name](form.params) }}
+              {% elif errors and errors[question.id] %}
```

Comment:
Let's encapsulate this logic common to multiquestions and questions in a macro, we can define one at the top of this file https://jinja.palletsprojects.com/en/2.11.x/templates/#macros